service/version_bundles.go

```diff
@@ -1,346 +1,23 @@
-				{
-					Name:    "calico",
-					Version: "2.6.2",
-				},
-				{
-					Name:    "docker",
-					Version: "1.12.6",
-				},
-				{
-					Name:    "etcd",
-					Version: "3.2.7",
-				},
-				{
-					Name:    "kubedns",
-					Version: "1.14.5",
-				},
-				{
-					Name:    "kubernetes",
-					Version: "1.8.1",
-				},
-				{
-					Name:    "nginx-ingress-controller",
-					Version: "0.9.0",
-				},
-			},
-			Dependencies: []versionbundle.Dependency{},
-			Deprecated:   true,
-			Name:         "kvm-operator",
-			Time:         time.Date(2017, time.October, 26, 16, 38, 0, 0, time.UTC),
-			Version:      "0.1.0",
-			WIP:          false,
-		},
-		{
-			Changelogs: []versionbundle.Changelog{
-				{
-					Component:   "kubernetes",
-					Description: "Updated to kubernetes 1.8.4. Fixes a goroutine leak in the k8s api.",
-					Kind:        versionbundle.KindChanged,
-				},
-			},
-			Components: []versionbundle.Component{
-				{
-					Name:    "calico",
-					Version: "2.6.2",
-				},
-				{
-					Name:    "docker",
-					Version: "1.12.6",
-				},
-				{
-					Name:    "etcd",
-					Version: "3.2.7",
-				},
-				{
-					Name:    "kubedns",
-					Version: "1.14.5",
-				},
-				{
-					Name:    "kubernetes",
-					Version: "1.8.4",
-				},
-				{
-					Name:    "nginx-ingress-controller",
-					Version: "0.9.0",
-				},
-			},
-			Dependencies: []versionbundle.Dependency{},
-			Deprecated:   true,
-			Name:         "kvm-operator",
-			Time:         time.Date(2017, time.December, 12, 10, 00, 0, 0, time.UTC),
-			Version:      "1.0.0",
-			WIP:          false,
-		},
-		{
-			Changelogs: []versionbundle.Changelog{
-				{
-					Component:   "kubernetes",
-					Description: "Enable encryption at rest",
-					Kind:        versionbundle.KindChanged,
-				},
-			},
-			Components: []versionbundle.Component{
-				{
-					Name:    "calico",
-					Version: "2.6.2",
-				},
-				{
-					Name:    "docker",
-					Version: "1.12.6",
-				},
-				{
-					Name:    "etcd",
-					Version: "3.2.7",
-				},
-				{
-					Name:    "kubedns",
-					Version: "1.14.5",
-				},
-				{
-					Name:    "kubernetes",
-					Version: "1.8.4",
-				},
-				{
-					Name:    "nginx-ingress-controller",
-					Version: "0.9.0",
-				},
-			},
-			Dependencies: []versionbundle.Dependency{},
-			Deprecated:   true,
-			Name:         "kvm-operator",
-			Time:         time.Date(2017, time.December, 19, 10, 00, 0, 0, time.UTC),
-			Version:      "1.1.0",
-			WIP:          false,
-		},
-		{
-			Changelogs: []versionbundle.Changelog{
-				{
+	var versionBundles []versionbundle.Bundle
+
+	versionBundles = append(versionBundles, v2.VersionBundles()...) // NOTE this is special because it was created during the introduction of version bundles.
+	versionBundles = append(versionBundles, v3.VersionBundle())
+	versionBundles = append(versionBundles, v4.VersionBundle())
+	versionBundles = append(versionBundles, v5.VersionBundle())
+	versionBundles = append(versionBundles, v6.VersionBundle())
```

Comment:
This is a thing we have to ensure manually for now but we can also automate this in the future.